### PR TITLE
Example for Mu law encoding  

### PR DESCRIPTION
**Pull request by harishsdev**
Example for Mu law encoding

### CODE REVIEW

**Comment by harishsdev** [comment]
PR for Issue https://github.com/pytorch/audio/issues/1564

**Comment by harishsdev** [comment]
@mthrok please help to review

**Comment by harishsdev** [comment]
@nateanl please help to review

File: torchaudio/transforms.py [hunk 1]
```diff
@@ -604,6 +604,13 @@ class MuLawEncoding(torch.nn.Module):
 
     Args:
         quantization_channels (int, optional): Number of channels. (Default: ``256``)
+
+    Example
+
+       >>> waveform, sample_rate = torchaudio.load('test.wav', normalization=True)
```
Comment (nateanl) [inline torchaudio/transforms.py:610]: nit: ``normalization`` is changed to ``normalize`` now. Let's replace it. 
```suggestion
       >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
```

File: torchaudio/transforms.py [hunk 2]
```diff
@@ -604,6 +604,13 @@ class MuLawEncoding(torch.nn.Module):
...
+       >>> transform = torchaudio.transforms.MuLawEncoding(quantization_channels = 512)
```
Comment (nateanl) [inline torchaudio/transforms.py:611]: nit: remove spaces.
```suggestion
       >>> transform = torchaudio.transforms.MuLawEncoding(quantization_channels=512)
```

**Review by nateanl — APPROVED** [review]
LGTM. Thanks for the change @harishsdev. We can merge it after fixing all nits!

File: torchaudio/transforms.py [hunk 3]
```diff
@@ -604,6 +604,13 @@ class MuLawEncoding(torch.nn.Module):
 
     Args:
         quantization_channels (int, optional): Number of channels. (Default: ``256``)
+
+    Example
+
```
Comment (nateanl) [inline torchaudio/transforms.py:609]:
```suggestion
```

**Comment by harishsdev** [comment]
@nateanl  thanks for merging it
